lib/view/templates.py

```diff
@@ -2571,3 +2571,216 @@ def format_asinfo_error(msg: str):
     from_source=("data",),
     order_by=(FieldSorter("Node"), FieldSorter("Client Version")),
 )
+
+# MRT Monitor metrics template
+info_transactions_monitors_sheet = Sheet(
+    (
+        namespace_field,
+        node_field,
+        hidden_node_id_field,
+        # Monitor Counts
+        Field(
+            "Count",
+            Projectors.Number("ns_stats", "mrt_monitors"),
+            converter=Converters.scientific_units,
+            aggregator=Aggregators.sum(),
```

Comment:
Are these sum aggregators necessary? There is only one field being used for most of these.